front/app/containers/Admin/pages-menu/containers/BottomInfoForm/index.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState, useEffect } from 'react';
+import { useTheme } from 'styled-components';
+
+// components
+import QuillMultilocWithLocaleSwitcher from 'components/UI/QuillEditor/QuillMultilocWithLocaleSwitcher';
+import Error from 'components/UI/Error';
+import SectionFormWrapper from '../../components/SectionFormWrapper';
+import { Box, Button } from '@citizenlab/cl2-component-library';
+
+// i18n
+import { InjectedIntlProps } from 'react-intl';
+import { injectIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+// typings
+import { Multiloc, CLError } from 'typings';
+
+// constants
+import { pagesAndMenuBreadcrumb, homeBreadcrumb } from '../../constants';
+
+// resources
+import useHomepageSettings from 'hooks/useHomepageSettings';
+import { isNilOrError } from 'utils/helperUtils';
+import { isCLErrorJSON } from 'utils/errorUtils';
+import { updateHomepageSettings } from 'services/homepageSettings';
+
+const BottomInfoForm = ({ intl: { formatMessage } }: InjectedIntlProps) => {
+  const homepageSettings = useHomepageSettings();
+  const theme: any = useTheme();
+
+  const [bottomInfoSectionMultilocState, setBottomInfoSectionMultilocState] =
+    useState<Multiloc | null | undefined>(null);
+  const [isLoading, setIsLoading] = useState(false);
+  const [apiErrors, setApiErrors] = useState<CLError[] | null>(null);
+
+  useEffect(() => {
+    if (!isNilOrError(homepageSettings)) {
+      setBottomInfoSectionMultilocState(
+        homepageSettings.data.attributes.bottom_info_section_multiloc
+      );
+    }
+  }, [homepageSettings]);
+
+  const handleCustomSectionMultilocOnChange = (
+    homepageInfoPageMultiloc: Multiloc
+  ) => {
+    setBottomInfoSectionMultilocState(homepageInfoPageMultiloc);
+  };
+
+  const onSave = async () => {
+    setIsLoading(true);
+    try {
+      if (bottomInfoSectionMultilocState) {
+        await updateHomepageSettings({
+          bottom_info_section_multiloc: bottomInfoSectionMultilocState,
+        });
+      }
+      setIsLoading(false);
+    } catch (error) {
+      if (isCLErrorJSON(error)) {
+        setApiErrors(error.json.errors);
+      } else {
+        setApiErrors(error);
+      }
+      setIsLoading(false);
+    }
+  };
+
+  return (
+    <SectionFormWrapper
+      breadcrumbs={[
+        {
+          label: formatMessage(pagesAndMenuBreadcrumb.label),
+          linkTo: pagesAndMenuBreadcrumb.linkTo,
+        },
+        {
+          label: formatMessage(homeBreadcrumb.label),
+          linkTo: homeBreadcrumb.linkTo,
+        },
+        { label: formatMessage(messages.bottomInfoPageTitle) },
+      ]}
+      title="Bottom Info Section"
+      stickyMenuContents={
+        <Button disabled={isLoading} onClick={onSave}>
+          Save Bottom Info Form
```

Comment:
It's tempting to not put them there from the start because it's faster, but it's a matter of time before we forget it and some copy is not translated. So I would put it in messages files from the start. They're pretty easy to change afterwards. If that's not clear, I'd be happy to explain.